images/gitlab/main.tf

```diff
@@ -0,0 +1,48 @@
+terraform {
+  required_providers {
+    oci = { source = "chainguard-dev/oci" }
+  }
+}
+
+variable "target_repository" {
+  description = "The docker repo into which the image and attestations should be published."
+}
+
+locals {
+  components = toset(["kas"])
+}
+
+module "config" {
+  for_each = local.components
+  source   = "./config"
+  name     = each.key
+}
+
+module "latest" {
+  for_each = local.components
+  source   = "../../tflib/publisher"
+
+  name               = basename(path.module)
+  target_repository  = "${var.target_repository}-${each.key}"
+  config             = module.config[each.key].config
+  extra_dev_packages = []
```

Comment:
```suggestion
  build-dev = true
```

Reply:
True! The extra_dev_packages was empty now.